backend/infrahub/message_bus/operations/event/worker.py

```diff
@@ -2,11 +2,10 @@
 
 from infrahub.message_bus import messages
 from infrahub.services import InfrahubServices
-from infrahub.workflows.catalogue import WEBHOOK_CONFIGURE
 
 
 @flow(name="event-worker-newprimary-api")
 async def new_primary_api(message: messages.EventWorkerNewPrimaryAPI, service: InfrahubServices) -> None:
     service.log.info("api_worker promoted to primary", worker_id=message.worker_id)
 
-    await service.workflow.submit_workflow(workflow=WEBHOOK_CONFIGURE)
+    # NOTE Should we remove this message ?
```

Comment:
@ogenstad what do you think ?

Reply:
The reason to have this is to ensure that the redis cache gets populated primarily on server startup if the system has been done for some time. It's to avoid a situation where the webhooks aren't working because they don't exist in the cache. With the changes you made with regards to just checking the cache to see if it exists there before loading the config from the SDK I don't see that we need to keep any of this here.